### PR TITLE
Back port of poc fixes into develop

### PR DESCRIPTION
**Pull request by ErikApption**
- Python fixes 
- Minor url fixes

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Unit test (new or update to tests)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (change that would cause documentation to be updated)
- [ ] Configuration change (change that would affect the configuration of the application)

## Checklist:

- [X] PR is submitted to the correct branch `develop`.
- [X] Code follows the code style of this project.
- [ ] Changes are covered by Specflow tests and all new or existing tests are passing.
- [ ] Any new or updated translatable strings have been added to the localization files.
- [ ] Necessary and relevant documentation has been created or updated.
- [ ] There are either no changes to the application's configuration or the necessary changes in the infrastructure repository are included in a separate PR.

### CODE REVIEW

**Review by vb-ssc — APPROVED** [review]
Looks good to me

File: .gitignore [hunk 1]
```diff
@@ -63,3 +63,4 @@ Portal/src/Datahub.Core/Utils/Certs/gc1.pem
 Portal/src/Datahub.Portal/MessageFolder/
 ResourceProvisioner/test/ResourceProvisioner_PyFunctions_Tests/venv/*
 /Portal/utils/JsonTranslator/Properties/launchSettings.json
+/Portal/src/Datahub.Portal/MissingTranslations-default..json
```
Comment (yjmrobert) [inline .gitignore:66]: We don't need this file do we?

File: ResourceProvisioner/src/ResourceProvisioner_PyFunctions/function_app.py [hunk 2]
```diff
@@ -16,8 +16,8 @@
 
 def get_config():
     asb_connection_str = os.getenv('DatahubServiceBus')
-    queue_name = "bug-report" // os.getenv('AzureServiceBusQueueName4Bugs')
-    check_results_queue_name = "infrastructure-health-check-results" // os.getenv('AzureServiceBusQueueName4Results')
+    queue_name = os.getenv('AzureServiceBusQueueName4Bugs') or "bug-report"
+    check_results_queue_name = os.getenv('AzureServiceBusQueueName4Results') or "infrastructure-health-check-results"
```
Comment (yjmrobert) [inline ResourceProvisioner/src/ResourceProvisioner_PyFunctions/function_app.py:20]: Is the 4 an abbreviation for "for"?

File: .gitignore (hunk 1 above)
Comment (ErikApption) [reply .gitignore:66]: we should ignore this file

File: ResourceProvisioner/src/ResourceProvisioner_PyFunctions/function_app.py (hunk 2 above)
Comment (vb-ssc) [reply ResourceProvisioner/src/ResourceProvisioner_PyFunctions/function_app.py:20]: yes sir!

**Pull request merged**